src/yuzu/configuration/configure_motion_touch.cpp

```diff
@@ -282,32 +324,31 @@ void ConfigureMotionTouch::ApplyConfiguration() {
         return;
     }
 
-    std::string motion_engine = ui->motion_provider->currentData().toString().toStdString();
     std::string touch_engine = ui->touch_provider->currentData().toString().toStdString();
 
-    Common::ParamPackage motion_param{}, touch_param{};
-    motion_param.Set("engine", std::move(motion_engine));
+    Common::ParamPackage  touch_param{};
     touch_param.Set("engine", std::move(touch_engine));
 
-    if (motion_engine == "motion_emu") {
-        motion_param.Set("sensitivity", static_cast<float>(ui->motion_sensitivity->value()));
-    }
-
     if (touch_engine == "cemuhookudp") {
         touch_param.Set("min_x", min_x);
         touch_param.Set("min_y", min_y);
         touch_param.Set("max_x", max_x);
         touch_param.Set("max_y", max_y);
     }
 
-    Settings::values.motion_device = motion_param.Serialize();
+    std::string input_servers="";
+    // Concatenate all servers
+    for (auto item : udp_server_list_model->stringList()) {
+        input_servers += item.toStdString()+",";
+    }
+    // Remove last coma
+    input_servers = input_servers.substr(0, input_servers.size() - 1);
```

Comment:
```suggestion
    input_servers = input_servers.substr(0, input_servers.size() - 1);
```

This can be converted to `input_servers.pop_back();`

You don't need to create another substring.